app/views/hearings/_court_applications.html.haml

```diff
@@ -0,0 +1,61 @@
+%h2.govuk-heading-m
+  = t('hearings.show.court_applications.heading')
+- if hearing&.court_applications.present?
+  %div.govuk-accordion#accordion-with-summary-sections{:data => {:module => "govuk-accordion"}}
```

Comment:
🤔 be nice to have a way to abstract all this repetition for the accordion out to a helper/presenter

Reply:
@jsugarman, if you are ok with this, I would prefer to tackle this as a separate ticket/pr so as not to delay implementing court applications.